src/NoteMouseHandler/SelectionMouseHandler.js

```diff
@@ -1,72 +1,71 @@
 import _ from "lodash"
-import MouseHandler, { defaultActionFactory, getLocal } from "./NoteMouseHandler"
+import MouseHandler, { defaultActionFactory } from "./NoteMouseHandler"
 import { createNoteAction } from "./PencilMouseHandler"
 import Rect from "../model/Rect"
 import { pointSub } from "../helpers/point"
 
+const defaultCursor = "crosshair"
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->